src/data/polynomial/eval.lean

```diff
@@ -81,11 +81,10 @@ by rw [bit0, eval₂_add, bit0]
 by rw [bit1, eval₂_add, eval₂_bit0, eval₂_one, bit1]
 
 @[simp] lemma eval₂_smul (g : R →+* S) (p : polynomial R) (x : S) {s : R} :
-  eval₂ g x (s • p) = g s • eval₂ g x p :=
```

Comment:
Note that if this is generalized to an `R'` that satisfies `is_scalar_tower` or similar, then the original formulation makes more sense after all.

Reply:
I'm not sure I understand you. What would be an example of a situation including `is_scalar_tower` for which the original formulation makes more sense?

Reply:
You're right, I was imagining a situation where `eval₂` was defined as `p.sum (λ e a, f a • x ^ e)` or something, but I don't think that's worth worrying about.